### DISCUSSION TITLE
SyntaxError: Cannot use import statement outside a module

### DISCUSSION THREAD

**Discussion by mihaimazareanu** [neuefische/questions#366] [Web] [answered]
Since I imported `ReactPasswordChecklist` in my app, to measure the strength of a password, I keep getting this error:

    SyntaxError: Cannot use import statement outside a module

I researched a bit and the proposed solution is to add in `package.json` document `"type": "module"`

However, when I do that, I get another error:

    TypeError: Cannot read properties of undefined (reading 'withConfig')

No matter what I do, I can't seem to be able to fix this problem and Vercel won't deploy the preview...

Any idea?  🤔 

 I imported `ReactPasswordChecklist` like this:

    import ReactPasswordChecklist from "react-password-checklist";

Here is the PR, if you would like to look at the whole code: https://github.com/mihaimazareanu/capstone-project-kidsfi/pull/14

**Comment by actyralf** [comment, marked as answer]
Seems to be a problem with server side rendering (SSR) and `react-password-checklist`.
You should be able to solve this by importing the module only on client side:

```
import dynamic from 'next/dynamic';
const ReactPasswordChecklist = dynamic(() => import('react-password-checklist'), {
  ssr: false,
});
```
Further info: https://github.com/sators/react-password-checklist/issues/18

**Reply by mihaimazareanu** [reply]
Thank you, Ralf, this has solved the issue. And thank you for doing this on a Sunday... 🙏
